packages/kbn-securitysolution-list-constants/index.ts

```diff
@@ -70,30 +72,76 @@ export const MAXIMUM_SMALL_VALUE_LIST_SIZE = 65536;
 
 export const MAXIMUM_SMALL_IP_RANGE_VALUE_LIST_DASH_SIZE = 200;
 
-/** ID of trusted apps agnostic list */
-export const ENDPOINT_TRUSTED_APPS_LIST_ID = 'endpoint_trusted_apps';
+/**
+ * List definitions for Endpoint Artifact
+ */
+export const ENDPOINT_ARTIFACT_LISTS = deepFreeze({
```

Comment:
Going forward, we should only use this `const` and no longer use the individual ones further below. I have marked those below as `@deprecated`, so hopefully your IDE's will remind you if you attempt to still use them.

Reply:
awesome idea, thank you! 🎉

Reply:
Thanks for doing this!